### PR TITLE
CMPS-157

### PR DESCRIPTION
**Pull request by YuriiFetko**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,5 @@
 debug.log
 
 logs/
+/Src/Campus.Master.API/UI/angular.json
```
Comment (profileEnumerable) [inline .gitignore]: Undo changes for this file

File: Src/Campus.Master.API/UI/src/app/modules/calendar/calendar.component.sass [hunk 2]
```diff
@@ -1,11 +1,80 @@
+@import "src/styles/styles"
```
Comment (profileEnumerable) [inline Src/Campus.Master.API/UI/src/app/modules/calendar/calendar.component.sass:1]: Consider replacement to import alias @import "styles"

**Review by profileEnumerable — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (antonAce) [reply .gitignore]: Done

File: Src/Campus.Master.API/UI/src/app/modules/calendar/calendar.component.sass (hunk 2 above)
Comment (antonAce) [reply Src/Campus.Master.API/UI/src/app/modules/calendar/calendar.component.sass]: Done

**Review by antonAce — APPROVED** [review]
:shipit:

**Review by profileEnumerable — APPROVED** [review]

**Pull request merged**